node/src/neighborhood/mod.rs

```diff
@@ -4660,72 +4662,41 @@ mod tests {
                         vec![make_node_descriptor(make_ip(1))],
                         rate_pack(100),
                     ),
-                    min_hops_count: MIN_HOPS_COUNT_FOR_TEST,
+                    min_hops: min_hops_in_neighborhood,
                 },
                 make_wallet("earning"),
                 None,
                 test_name,
             ),
         );
+        subject.logger = Logger::new(test_name);
         subject.persistent_config_opt = Some(Box::new(
             PersistentConfigurationMock::new()
-                .min_hops_count_result(Err(PersistentConfigError::NotPresent)),
+                .min_hops_params(&min_hops_params_arc)
```

Comment:
No need for `.min_hops_params(...)`, the assertion is now done by different means, and this one is just bloating code.  The fact that `.min_hops_result(Ok(min_hops_in_db))` sets a certain result which also finds the assertion message in the survey of the actor's guts doesn't actually need any further extensions.

Reply:
Done 👍